src/styles/blocks/card.scss

```diff
@@ -0,0 +1,123 @@
+.card {
+  @include flex-column;
+
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  padding-block: 32px 16px;
```

Comment:
![image](https://github.com/mate-academy/layout_catalog/assets/96265844/55305d1a-e4c3-442e-938c-15941cd9dd15)
it must be a common card paddings